src/absil/il.fs

```diff
@@ -18,6 +18,7 @@ open System.Collections
  
 let logging = false 
 
+#if FX_RUNNING_ON_MONO
```

Comment:
Why not use "runningOnMono = false" for CoreCLR?  That will reduce the later diff won't it?